Users/service.js

```diff
@@ -1,51 +1,63 @@
 var response = require('../Objects/response.js');
 
-var createUser = function (json) {
-    // Creates a user
-
-    // TODO: Create user and remove console.log
-    console.log(json);
+var createUser = (json) => {
+	// Create Student
+	if (json.role == 'Student') {
+		var Student = new StudentModel({
+			approvalStatus: json.approvalStatus,
+			email: json.email,
+			firstName: json.firstName,
+			lastName: json.lastName,
+			points: json.points,
+			role: json.role,
+			studentId: json.studentId
+		});
+	// TODO: Write new Student to database
+	}
+
+	// Create Admin
+	else if (json.role == 'Admin') {
```

Comment:
There should be no line breaks between the closing bracket of the ```if``` and the ```else if (...) {```: https://google.github.io/styleguide/jsguide.html#formatting-nonempty-blocks